server/src/entities/system-metadata.entity.ts

```diff
@@ -22,6 +23,7 @@ export type VersionCheckMetadata = { checkedAt: string; releaseVersion: string }
 
 export interface SystemMetadata extends Record<SystemMetadataKey, Record<string, any>> {
   [SystemMetadataKey.REVERSE_GEOCODING_STATE]: { lastUpdate?: string; lastImportFileName?: string };
+  [SystemMetadataKey.FACIAL_RECOGNITION_STATE]: { lastRun?: Date };
```

Comment:
I think using a date here is just going to cause problems. This is serialized when it is written, and probably deserializes to a string when it is read afterwards. Did you check this behavior?

Reply:
I didn't check that it's a date - I think you're right that it's a string in practice.